README.md

```diff
@@ -6,11 +6,30 @@
 
 A _small_ Docker image for [composer](https://getcomposer.org), a dependency management tool for PHP.
 
-## Dockerfile Links
+## Images
 
-* `php-7.1`, `php-7`, `latest` ([php-7.1/Dockerfile](https://github.com/graze/docker-composer/blob/master/php-7.1/Dockerfile))
+This repository generates an image for each composer version, php version and combination there of
+
+The available composer versions are:
+
+* `1.4.1`, `latest`
+* `1.4.0`
+* `1.3.3`
+* `1.3.2`
+* `1.3.1`
+* `1.3.0`
```

Comment:
Nothing specific, 1.4 is relatively recent, 1.2 was quite a long time ago. Seemed to be a good compromise.